tools/test/audio/src_run.sh

```diff
@@ -10,15 +10,12 @@ FN_IN=$5
 FN_OUT=$6
 
 # The HOST_ROOT path need to be retrived from SOFT .configure command
-HOST_ROOT=../../../host-root
+HOST_ROOT=../../../../host-root
 HOST_EXE=$HOST_ROOT/bin/testbench
 HOST_LIB=$HOST_ROOT/lib
 
 # Topology
-INFMT=s${BITS_IN}le
-OUTFMT=s${BITS_IN}le
-MCLK=24576k
-TPLG=../../topology/test/test-playback-ssp2-mclk-0-I2S-${COMP}-${INFMT}-${OUTFMT}-48k-${MCLK}-nocodec.tplg
+TPLG=../../topology/sof-apl-src-pcm512x.tplg
```

Comment:
I need to switch back to test-playback topology in this script. The proposed topology doesn't work for 32 bit tests.

Reply:
Done. I will make another testbench specific PR next.